python/paddle/fluid/layers/nn.py

```diff
@@ -2044,6 +2044,7 @@ def pool2d(input,
            global_pooling=False,
            use_cudnn=True,
            ceil_mode=False,
+           exclusive=True,
```

Comment:
For  forward compatibility:

```
name=None,
exclusive=True):
```

Reply:
Thanks for your suggestion and have fixed for pool2d and pool3d